client/src/components/CreatePost.tsx

```diff
@@ -36,147 +36,148 @@ function CreatePost() {
+                label="Body"
+                data-testid="bodyTextField"
+                placeholder="description"
+                multiline
+                rows={6}
+                onChange={(e) => setForm({ ...form, body: e.target.value })}
+              />
+            </Grid>
+
+            <Grid item xs={12} md={4}>
+              <TextField
+                required
+                label="Upload Thumbnail"
+                type="file"
+                fullWidth
+                data-testid="fileField"
+                InputLabelProps={{ shrink: true }}
+                size="small"
+                onChange={(e) => handleImageUpload(e)}
+              />
+            </Grid>
+
+            <Grid item xs={12} md={3}>
+              <TextField
+                label="Event Capacity"
+                placeholder="40"
+                fullWidth
+                size="small"
+                onChange={(e) =>
+                  setForm({ ...form, eventCapacity: e.target.value })
+                }
+              />
             </Grid>
 
-            {/* form  done */}
-            <Box sx={{ mt: 2 }}>
-              <Divider />
-            </Box>
-
-            <Grid container spacing={2} justifyContent="center">
-              <Grid item xs={5} md={3}>
-                <Tooltip title="Enter all required fields!">
-                  <Box>
-                    <Button
-                      fullWidth
-                      variant="contained"
-                      sx={{ mt: 3, mb: 2 }}
-                      color="secondary"
-                      onClick={() => handleClickOpen()}
-                      data-testid="previewButton"
-                      size="large"
-                      disabled={!(title !== "" && body !== "")}
-                    >
-                      Preview
-                    </Button>
-                  </Box>
-                </Tooltip>
-              </Grid>
-              <Grid item xs={7} md={5}>
-                <Button
-                  fullWidth
-                  variant="contained"
-                  sx={{ mt: 3, mb: 2 }}
-                  size="large"
-                >
-                  Create
-                </Button>
-              </Grid>
+            <Grid item xs={12} md={5}>
+              <TextField
+                fullWidth
+                label="Location"
+                placeholder="Deerfield Hall"
+                size="small"
+                onChange={(e) =>
+                  setForm({ ...form, eventLocation: e.target.value })
+                }
+              />
             </Grid>
+
+            <Grid item xs={12}>
+              <TextField
+                fullWidth
+                label="Tags (Separate tags by a comma)"
+                placeholder="Clubs, Math, MCS"
+                size="small"
+                onChange={(e) => setForm({ ...form, tags: e.target.value })}
+              />
+            </Grid>
+          </Grid>
+
+          {/* form  done */}
```

Comment:
Is this comment in the correct place?

Reply:
by the comment, I meant the input fields or form were done, the next components were just the buttons